### PR TITLE
Fixes #1643 : Updated UI

### LINKED ISSUE #1643: Improve UI
**Actual Behaviour**

Containers are not been used Properly and many minor design Issues

**Expected Behaviour**

Beautiful Website

**Steps to reproduce it**

Fixing the UI

**Would you like to work on the issue?**

Yes

sched eventyay com_ pc

### PR DESCRIPTION
**Pull request by naveentherocker**
Fixes #1643 
Worked on the Initiative #1459 

Changes:
Added Container 
Added a Different Section for the `Apps with single sessions page`
Fixed a Minor Issue in Banner

Screenshots for the change: 
_c__users_naveen_documents_github_open-event-webapp_src_backend_overviewsite_index html pc

### CODE REVIEW

**Comment by naveentherocker** [comment]
@geekyd Fixed It.

**Comment by naveentherocker** [comment]
@mariobehling @hpdang @geekyd Please Have a look at it.

**Comment by naveentherocker** [comment]
Now it looks like this.
_c__users_naveen_documents_github_open-event-webapp_src_backend_overviewsite_index html pc 1

**Comment by geekyd** [comment]
@naveentherocker Nice work. Please make sure
- Both the headings are blue in color 
- Add border to event similar to eventyay

File: src/backend/overviewSite/index.html [hunk 1]
```diff
@@ -145,104 +146,103 @@
...
+            events <br> Feast your eyes on our beautiful example events.</p>
     </div>
     <div class="tint"></div>
 </div>
 
-<div class="container-fluid bg-3">
-    <h2 class="margin"> Apps with expandable sessions page </h2>
-    <div class="row text-center">
+<div class="container bg-3">
+    <h2 class="margin" align="left"> Apps with expandable sessions page </h2>
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: Remove `align="left"`

File: src/backend/overviewSite/index.html [hunk 2]
```diff
@@ -145,104 +146,103 @@
...
         </div>
     </div>
 </div>
 
-
-<div class="container-fluid bg-3">
-    <h2 class="margin">Apps with single sessions page</h2>
-    <div class="row text-center">
+<div class="container bg-1 text-center">
+    <h2 class="margin" align="left">Apps with single sessions page</h2>
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: Remove `align="left"`

**Comment by naveentherocker** [comment]
Fixed @geekyd 
_c__users_naveen_documents_github_open-event-webapp_src_backend_overviewsite_index html pc 2

File: src/backend/overviewSite/index.html [hunk 3]
```diff
@@ -101,15 +101,47 @@
...
         }
 
-        .margin {
-            margin-bottom: 50px;
+       
+        .event-holder .caption {
+            background-color: #FFFFFF;
+            transition: background-color 0.5s;
         }
+
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: remove extra lines.

File: src/backend/overviewSite/index.html [hunk 4]
```diff
@@ -101,15 +101,47 @@
...
+            max-height: 160px;
+            width: 100%;
+           
+            border: none;
+        }
+
+        .event-holder .thumbnail {
+            padding: 0;
+        }
+
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: remove extra lines, there should be one line

**Comment by geekyd** [comment]
@naveentherocker Can you also provide a deployment link? Upload overviewSite folder to gh-pages.

**Comment by naveentherocker** [comment]
@geekyd Deploy Link - https://naveentherocker.github.io/open-event-webapp/index.html

**Comment by naveentherocker** [comment]
@agbilotia1998 It works fine for me know. Screenshot Please

**Comment by naveentherocker** [comment]
@fossasia/open-event-web-app-admin Please Review

**Comment by geekyd** [comment]
@naveentherocker Please make sure that the images are of same size. Check the screen shot.
screen shot 2018-01-14 at 3 53 08 pm

**Comment by naveentherocker** [comment]
Fixed @geekyd

**Comment by geekyd** [comment]
@naveentherocker Deployment link please

**Comment by naveentherocker** [comment]
@geekyd Check the same link https://naveentherocker.github.io/open-event-webapp/index.html

File: src/backend/overviewSite/index.html [hunk 5]
```diff
@@ -100,15 +100,32 @@
         footer p {
             margin: 0px;
         }
+        .event-holder {
+            cursor: pointer;
 
-        img {
-            height: 160px;
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: @naveentherocker Do not remove this, The previous dimensions for image was good.

Comment (naveentherocker) [reply src/backend/overviewSite/index.html]: @geekyd The 2nd Image seems to be not perfect when it is `160px`. You face the earlier issue when it was in `160px`

Comment (geekyd) [reply src/backend/overviewSite/index.html]: 140px is not a good size. Lets not remove it.

Comment (naveentherocker) [reply src/backend/overviewSite/index.html]: @geekyd Now fine. All the Images are `160px`

File: src/backend/overviewSite/index.html [hunk 6]
```diff
@@ -100,15 +100,33 @@
...
-            width: 310px;
-            max-width: 100%;
         }
-
-        .margin {
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: @naveentherocker Revet this, this class is used in the headers.

Comment (naveentherocker) [reply src/backend/overviewSite/index.html]: Done @geekyd

**Comment by naveentherocker** [comment]
@geekyd Please Merge this If Possible

**Comment by naveentherocker** [comment]
@geekyd Codacy is not Running. So it would be great If you could Review it.

**Comment by naveentherocker** [comment]
It is failing because of this Review Change https://github.com/fossasia/open-event-webapp/pull/1644/files/b89a18e9c7e7c7fa8ef6f649610a43cec8991bc9#diff-90031741b8737d92d679107f472a687c

**Comment by naveentherocker** [comment]
@geekyd Now it is Ready for the Merge.

**Pull request merged**